app/services/onestop_id.rb

```diff
@@ -3,6 +3,9 @@
 class OnestopId
   ENTITY_TO_PREFIX = {
     'stop' => 's',
+    'stopstation' => 's',
+    'stopplatform' => 's',
+    'stopentrance' => 's',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.